### DISCUSSION TITLE
[Feature] Download status

### DISCUSSION THREAD

**Discussion by Pelukosa** [immich-app/immich#12236] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to be able to see the status of the download from the mobile application.

When I want to download a video that is only on the server to my phone, if it is very large, for example 2gb, it would be nice to see some kind of status, such as when you download a file on android, which appears in the notification bar a progress bar.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe: #10100

Like the template requests, please search for existing FRs before opening a new one.
